src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -1484,100 +1506,76 @@ private static bool IsInvalidValue(ref float value)
 
         /// This will not be public opened.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        public new void Dispose()
```

Comment:
could you check this Mr. Lee's comment again?
https://github.com/Samsung/TizenFX/pull/6727#discussion_r1988470751

Reply:
In this patch, we will not do anything in the Dispose of Color, because the SwigCPtr in Color will not be created.
So it doesn't has issue of multi thread.